pkg/auxnetdevice/auxNetResourcePool.go

```diff
@@ -60,3 +64,8 @@ func (ap *auxNetResourcePool) GetDeviceSpecs(deviceIDs []string) []*pluginapi.De
 	}
 	return devSpecs
 }
+
+// GetPoolKind returns device kind for CDI spec
```

Comment:
nit: i dont think its relevant here that CDI uses this information.

lets name this method: GetPoolType which will essentially return the type of devices used by the  pool WDYT ?

Reply:
What do you mean? could you provide more details?